### PR TITLE
Remove use of unexisting / useless variables

### PR DESCRIPTION
**Pull request by Tithugues**
### Description

Remove use of unexisting variables
Remove useless variables

### CODE REVIEW

File: libraries/classes/Plugins/Export/Helpers/Pdf.php [hunk 1]
```diff
@@ -593,13 +589,13 @@ public function getTableDef(
          * Presently, this information is not used. We will have to find out
          * way of displaying it.
          */
-        $unique_keys = [];
+        /*$unique_keys = [];
```
Comment (mauriciofauth) [inline libraries/classes/Plugins/Export/Helpers/Pdf.php]: Could you remove the commented code?

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

File: libraries/classes/Plugins/Export/Helpers/Pdf.php (hunk 1 above)
Comment (Tithugues) [reply libraries/classes/Plugins/Export/Helpers/Pdf.php]: As you can see in the comment above, it was expected to find a way to display the information.

Can you please confirm that you want me to remove the 2 comments?

Thank you.

Comment (mauriciofauth) [reply libraries/classes/Plugins/Export/Helpers/Pdf.php]: Apparently, this piece of code was added in #1627 and was never used. I think is safe to remove it.

Comment (mauriciofauth) [reply libraries/classes/Plugins/Export/Helpers/Pdf.php]: Maybe we can open an issue about it, I don't know. An issue is better than a comment on the code.

Comment (Tithugues) [reply libraries/classes/Plugins/Export/Helpers/Pdf.php]: Comment removed.

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**